packages/data/tsconfig.json

```diff
@@ -14,7 +14,5 @@
 		{ "path": "../priority-queue" },
 		{ "path": "../redux-routine" }
 	],
-	"include": [
-		"src/**/*"
-	]
+	"include": [ "src/**/*" ]
```

Comment:
This too.

Reply:
@kevin940726 I am not sure about it, how do I need to fix it? It looks same to me https://github.com/WordPress/gutenberg/blob/trunk/packages/data/tsconfig.json

Reply:
I think it's auto-formatted once you saved this file in the editor. You can also just revert this back to its original formatting too.